pkg/types/aws/defaults/platform.go

```diff
@@ -24,6 +25,11 @@ var (
 			// "us-east-1":      {"m6g.xlarge", "m6gd.xlarge"},
 		},
 	}
+	// us-east-1e is a well-known limited zone. Create base infra (networking),
+	// is useless as it does not offer supported instance types.
+	skippedZonesByRegion = map[string][]string{
+		"us-east-1": {"us-east-1e"},
+	}
```

Comment:
If you're going to use sets to exclude these zones, why not simplify this and just use a list?
```suggestion
	skippedZones = []string{"us-east-1e"}
```